ons_alpha/jinja2/templates/pages/bulletin_page.html

```diff
@@ -3,105 +3,138 @@
 {% from "components/panel/_macro.njk" import onsPanel %}
 {% from "components/table-of-contents/_macro.njk" import onsTableOfContents %}
 
-{% block main %}
-    <div class="ons-u-fs-m ons-u-mt-s ons-u-pb-xxs release__document-type">{{ page.document_type|default("Bulletin") }}</div>
-    <h1 class="ons-u-fs-xxxl ons-u-mb-m">
-        <span>{{ page.full_title }}</span>
-        {% if page.is_accredited %}
-            <a href="https://uksa.statisticsauthority.gov.uk/about-the-authority/uk-statistical-system/types-of-official-statistics/" class="national-statistics__link ons-u-ml-s">
-                <img src="https://cdn.ons.gov.uk/assets/images/ons-logo/kitemark/v2/uksa-kitemark-en.svg" alt="UK Statistics Authority Kitemark" class="osr__logo" width="90">
-            </a>
-        {% endif %}
-    </h1>
+{% block header_area %}
+    <div class="bulletin-header">
+        <div class="bulletin-header__content">
+            <div class="ons-container">
+                {% include "templates/components/navigation/breadcrumbs.html" %}
+                <p class="ons-u-fs-l bulletin-header__doc-type">{{ page.document_type|default("Analysis") }}</p>
+
+                <div class="ons-grid ons-grid--flex-gap">
+                    <div class="ons-grid__col ons-col-10@m ">
+                        <h1 class="ons-u-fs-xxxl bulletin-header__heading">
+                            {{ page.full_title }}
+                        </h1>
+                    </div>
+                    <div class="ons-grid__col ons-col-2@m bulletin-header__kitemark">
+                        {% if page.is_accredited %}
+                            <a href="https://uksa.statisticsauthority.gov.uk/about-the-authority/uk-statistical-system/types-of-official-statistics/" class="bulletin-header__accreditation-link">
+                                <img src="https://cdn.ons.gov.uk/assets/images/ons-logo/kitemark/v2/uksa-kitemark-en.svg" alt="UK Statistics Authority Kitemark" class="bulletin-header__accreditation-logo" width="90">
+                            </a>
+                        {% endif %}
+                    </div>
+                </div>
+            </div>
 
-    <div class="ons-grid ons-u-ml-no">
-        {% block release_meta %}
-            <div class="ons-grid__col ons-grid__col--sticky@m ons-col-4@m ons-col-12@s ons-u-p-no">
-                <span class="ons-u-fs-r--b">{{ _("Release date") }}:</span>
-                <span class="ons-u-nowrap">{{ page.release_date|date("j F Y") }}</span>
+            <div class="ons-container">
+                <p class="bulletin-header__summary">{{ page.summary }}</p>
             </div>
-            {% if page.next_release_date %}
-                <div class="ons-grid__col ons-col-8@m ons-col-12@s ons-u-p-no@xxs@m">
-                    <span class="ons-u-fs-r--b">{{ _("Next release") }}:</span>
-                    <span class="ons-u-nowrap">{{ page.next_release_date|date("j F Y") }}</span>
+
+            {% block release_meta %}
```

Comment:
@ababic I have kept this `release_meta` block in place as it surrounded the same information previously when it was lower down the page. However, I'm not sure we need it - the release pages have a system of extending a base, and overriding this block, but the bulletin page does not (at least currently).